backend/src/main/kotlin/io/zell/zdb/log/LogSearch.kt

```diff
@@ -23,7 +24,7 @@ class LogSearch (logPath: Path) {
             val entry = reader.next()
 
             if (entry.isApplicationEntry) {
-                val applicationEntry = entry.applicationEntry
+                val applicationEntry = entry.applicationEntry as SerializedApplicationEntry
```

Comment:
:question: Did this changed as well ? :thinking:

Reply:
Yes, there's serialized and unserialized. Since we are only reading from the log, log entries are serialized.